Add -fPIC to CFLAGS

-fPIC is needed to build shared library code.

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>

Index: b/src/Makefile
===================================================================
--- a/src/Makefile
+++ b/src/Makefile
@@ -9,6 +9,7 @@
 SHARED_LIBNAME_VERSION = $(SHARED_LIBNAME).$(VERSION)
 SHARED_LIBNAME_MAJOR = $(SHARED_LIBNAME).$(shell echo $(VERSION) | cut -f1 -d.)
 SHARED_LIBNAME_FLAGS = -shared -Wl,-soname,$(SHARED_LIBNAME_MAJOR)
+CFLAGS = -fPIC
 
 ifeq ($(BUILD_STATIC),yes)
   BUILD_RULES += lib_static
